configs/olcf/config.20170313.beta1.A_WCYCL1850S.ne30_oECv3_ICG.edison

```diff
@@ -51,7 +51,7 @@ baseDirectory = /dir/to/analysis/output
 # option:
```

Comment:
Could you add these, right before line 35:
# provide an absolute path to put HTML in an alternative location (e.g. a web
 # portal)
 # htmlSubdirectory = /ccs/proj/cli115/www/USERNAME/RUNNAME
htmlSubdirectory = html
```

Reply:
done